src/pages/index.js

```diff
@@ -38,8 +38,7 @@ const IndexPage = ({ data }) => {
         <JoinBannerContent>
           <div>JOIN OUR TEAM</div>
           <span>
-            subscribe to our newsletter to be notified when applications open
-            for spring 2021
+            Developer applications for spring 2021 open November 29th 2021
```

Comment:
so sorry i just noticed this but it should say speing 2022

Reply:
its too late you already approved